### ISSUE TITLE
Problem in training the model

### ISSUE THREAD

**Issue by abdelrahman-gaber** [chuanqi305/MobileNetv2-SSDLite#7]
Hi, 
I am trying to train the model with a face detection dataset "WIDER_FACE" following the SFD tricks with SSD. The changes I did to your model is that I used different layers for detection and different input image size 640x640. I also used the converted tensorflow model as pretrained model. However, while training, the validation accuracy is not increasing, it is oscillating around very low values (0.09 - 0.11) and never increase. 
Note that I did the same changes and used this dataset to train your MobileNetV1 model in MobileNet-SSD repo, and it was working fine, so after 5K iterations the validation accuracy was around 0.3. But following the same approach with MobileNetV2 model didn't help, and the validation accuracy stays very low for long time which indicates that the model is not learning. 
Here you can find all the files I used for training, in addition to the output log file of the training. 
 https://drive.google.com/drive/folders/1G3Kb81POy7kplCp3PMhzEGSJ9OHODuW6?usp=sharing

Thank you.
